file-formats/chkv/type/zif_aff_chkv_v1.intf.abap

```diff
@@ -34,6 +34,12 @@ INTERFACE zif_aff_chkv_v1
       "! <p class="shorttext">Greater Equal</p>
       "! Greater equal
       greater_equal        TYPE ty_option VALUE 'GE',
+      "! <p class="shorttext">Less Than</p>
+      "! Less than
+      less_than            TYPE ty_option VALUE 'LT',
+      "! <p class="shorttext">Less Equal</p>
+      "! Less equal
+      less_equal           TYPE ty_option VALUE 'LE',
```

Comment:
I think we should move these constant and the corresponding type to zif_aff_types_v1